tests/evhtests/l7_evh_crd_test.go

```diff
@@ -832,3 +832,76 @@ func TestHTTPRuleCreateDeleteForEvh(t *testing.T) {
 
 	TearDownIngressForCacheSyncCheck(t, modelName)
 }
+
+func TestHTTPRuleCreateDeleteWithPkiRefForEvh(t *testing.T) {
+	// ingress secure foo.com/foo /bar
+	// create httprule /foo, nothing happens
+	// create hostrule, httprule gets attached check on /foo /bar
```

Comment:
Comments seems to be confusing with test case is doing. it seems same is applicable to all added test cases.